FrontEnd/Modules/DynamicItems/Scripts/Fields.js

```diff
@@ -1511,14 +1511,30 @@ export class Fields {
                             }).data("kendoDialog");
 
                             // Trigger the OK button click when the user presses enter in the dialog.
-                            dialog.element.keyup((event) => {
-                                if (!event.key || event.key.toLowerCase() !== "enter" || $(event.target).closest(".k-grid-header").length || event.target.tagName === "TEXTAREA") {
-                                    return;
-                                }
+                            dialog.element.on("keyup", function(event) {
+                                if (!event.key || event.key.toLowerCase() !== "enter") return;
+
+                                // array of class names to ignore
+                                const ignoreClasses = [
+                                    "k-filter-menu-container"
+                                ];
+
+                                // check if any of these elements are visible
+                                const ignoreVisible = ignoreClasses.some(cls => $("." + cls + ":visible").length > 0);
```

Comment:
Can be written shorter.
```suggestion
                                const ignoreVisible = ignoreClasses.some(cls => $(`.${cls}:visible`).length);
```